web_monitoring/tests/test_differs.py

```diff
@@ -60,3 +64,55 @@ def test_get_visible_text():
     html = '<!--First comment--><h1>First Heading</h1><p>First paragraph.</p>'
     actual = wd._get_visible_text(html)
     assert actual == 'First Heading First paragraph.'
+
+
+# TODO: extend this to other html differs
```

Comment:
It makes sense to me to keep the tests that generate output for humans in a separate file from the fully automated ones that can FAIL. Maybe we can rename them later so this is more clear.